src/components/L2/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Icon, Row } from '@tlon/indigo-react';
+
+import './Card.scss';
+
+export interface CardProps {
+  icon?: typeof Icon;
+  title?: string;
+  subtitle?: string;
+  onClick?: () => void;
+  children: any[];
+}
+
+export default function Card({
+  icon,
+  title,
+  subtitle,
+  onClick,
+  children,
+}: CardProps) {
+  return (
+    <Box className="card" onClick={onClick}>
+      {(!!icon || !!title) && (
+        <Row>
+          {!!icon && icon}
+          {!!title && <div className="card-title">{title}</div>}
+        </Row>
+      )}
+      {!!subtitle && <div className="card-subtitle">{subtitle}</div>}
```

Comment:
Functionally it shouldn't make a difference, but given the expected values defined by the `CardProps` it appears that `!!` is extraneous here. This is certainly a nitpick, so optional to act on it, but a change would improve the readability of this section.

Reply:
I thought React had a problem with non-boolean values for conditional rendering